userland/applications/activity_monitor/AppDelegate.cpp

```diff
@@ -1,13 +1,18 @@
 #include "ViewController.h"
 #include <libui/AppDelegate.h>
+#include <libfoundation/AssetManager.h>
 
 class AppDelegate : public UI::AppDelegate {
 public:
     AppDelegate() = default;
     virtual ~AppDelegate() = default;
 
     LG::Size preferred_desktop_window_size() const override { return LG::Size(220, 210); }
-    const char* icon_path() const override { return "/res/icons/apps/activity_monitor.icon"; }
+    const char* icon_path() const override 
```

Comment:
While it's fine to keep for now, but it would be good to get rid of these `icon_path()` functions somehow. I didn't think on how to implement this, may be you can think of something.